apps/src/codebridge/FileBrowser/prompts/openNewFolderPrompt.tsx

```diff
@@ -18,9 +15,10 @@ import {EVENTS} from '@cdo/apps/metrics/AnalyticsConstants';
 type OpenNewFilePromptArgsType = {
   parentId: FolderId;
   appName: string;
-  dialogControl: DialogControlInterface;
+  dialogControl: Pick<DialogControlInterface, 'showDialog'>;
```

Comment:
the only part of the `dialogControl` that the `openNewFilePrompt` actually cares about is being able to call `showDialog`, so we simplify it.